### ISSUE TITLE
`elm-tooling install` fails with curl@8.6.0

### ISSUE THREAD

**Issue by nokazn** [elm-tooling/elm-tooling-cli#127]
### Description

`elm-tooling install` command seems to fail with curl@8.6.0 .

```sh
$ cat ./elm-tooling.json
{
    "tools": {
        "elm": "0.19.1"
    }
}

$ npx elm-tooling install
/home/nokazn/app/repro/elm-tooling.json
ERR! elm 0.19.1

elm 0.19.1
    < https://github.com/elm/compiler/releases/download/0.19.1/binary-for-linux-64-bit.gz
    > /home/nokazn/.elm/elm-tooling/elm/0.19.1/elm

    curl -#fLw "" https://github.com/elm/compiler/releases/download/0.19.1/binary-for-linux-64-bit.gz
    The above command exited with exit code 2:
    curl: option -#fLw: blank argument where content is expected
    curl: try 'curl --help' for more information

$ curl --version
curl 8.6.0 (x86_64-pc-linux-gnu) libcurl/8.6.0 OpenSSL/3.2.1 zlib/1.3.1 brotli/1.1.0 zstd/1.5.5 libidn2/2.3.7 libpsl/0.21.2 libssh2/1.11.0 nghttp2/1.60.0 nghttp3/1.2.0
Release-Date: 2024-01-31
Protocols: dict file ftp ftps gopher gophers http https imap imaps ipfs ipns mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTP3 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM PSL SPNEGO SSL threadsafe TLS-SRP UnixSockets zstd
```

I think `-w ""` option for curl in the below line (which seems to be introduced in #62) is not supported from curl@8.6.0 .

https://github.com/elm-tooling/elm-tooling-cli/blob/5582d73e52ab8aa01ed12db16db1c9a24aad1830/src/commands/Install.ts#L553-L556

curl wouldn't fail without this `-w ""` option even in 8.6.0, and the current installation command succeeds with curl 8.**5**.0 .

I wondered I could contribute to fix this firstly, but have no idea to address this. 😢
Thank you.

**Comment by lydell** [comment]
Hi! Thanks for reporting!

Tricky problem indeed. Worst case we can get rid of `-w ""` – it’s better if it works for the majority (people on the latest version), than a minority (people with a “weird” .curlrc setting).

It would also be interesting to know if this was an intended change from curl’s side.

**Comment by cbrake** [comment]
Also seeing this on Arch Linux

**Comment by cbrake** [comment]
confirmed curl 8.5.0 works here -- will revert to that for now -- let me know if I can do anything to test/debug.

**Comment by lydell** [comment]
`-w "%{stderr}"` seems to have the same effect as `-w ""`, can you try if it works on curl 8.6.0? I don’t have the ability to test 8.6.0 right now. (Need to test Windows later, too.)

```
❯ echo '-w "hello from curlrc"' > ~/.curlrc

❯ curl google.com
<HTML><HEAD><meta http-equiv="content-type" content="text/html;charset=utf-8">
<TITLE>301 Moved</TITLE></HEAD><BODY>
<H1>301 Moved</H1>
The document has moved
<A HREF="http://www.google.com/">here</A>.
</BODY></HTML>
hello from curlrc⏎

❯ curl google.com -w "%{stderr}"
<HTML><HEAD><meta http-equiv="content-type" content="text/html;charset=utf-8">
<TITLE>301 Moved</TITLE></HEAD><BODY>
<H1>301 Moved</H1>
The document has moved
<A HREF="http://www.google.com/">here</A>.
</BODY></HTML>
```

Oh, and this is the curl commit that introduced the change btw: https://github.com/curl/curl/commit/07bcae89d5d0090f1d11866d5f9c98c3720a2838

**Comment by lydell** [comment]
Fix released in v1.15.1.

**Comment by cbrake** [comment]
works great here, thanks!

```
[cbrake@quark frontend]$ npx elm-tooling install
/scratch/simpleiot/go/frontend/elm-tooling.json
100% elm 0.19.1
100% elm-format 0.8.7
100% elm-json 0.2.13
100% elm-test-rs 3.0.0
elm 0.19.1: all good
elm-format 0.8.7: all good
elm-json 0.2.13: all good
elm-test-rs 3.0.0: all good
[cbrake@quark frontend]$ curl --version
curl 8.6.0 (x86_64-pc-linux-gnu) libcurl/8.6.0 OpenSSL/3.2.1 zlib/1.3.1 brotli/1.1.0 zstd/1.5.5 libidn2/2.3.7 libpsl/0.21.2 libssh2/1.11.0 nghttp2/1.60.0 nghttp3/1.2.0
Release-Date: 2024-01-31
Protocols: dict file ftp ftps gopher gophers http https imap imaps ipfs ipns mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTP3 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM PSL SPNEGO SSL threadsafe TLS-SRP UnixSockets zstd
```
